src/menu/menu_state.h

```diff
@@ -87,12 +90,17 @@ typedef struct {
         bool go_left;
         bool go_right;
         bool go_fast;
+        bool go_c_left;
```

Comment:
any reason that this is different from `go_up`, `go_down` etc., as the function is the same for the rest of the menu (was maybe a merge confiict).